### ISSUE TITLE
Latest version not published to Bower

### ISSUE THREAD

**Issue by willisplummer** [stripe/jquery.payment#240]
We're pegged to v ~1.4.3 via Bower, but we weren't getting behavior consistent with the latest change set. Some Visa cards were still being wrongly identified as ELO. It appears that the latest version of this library was published to NPM but not to Bower.

**Comment by jenanwise** [comment]
Hi @willisplummer! I don't know anything about bower, but `bower info jquery.payments` shows 1.4.3 released and that's our latest version. Are you sure this isn't a cacheing issue on your end? Can you produce a diff of what's in bower vs what's in npm?

**Comment by willisplummer** [comment]
Hi @jenanwise! I just cleared bower's cache and downloaded 1.4.3 to a new repo. Package.json reflects that the version number is `1.4.3` but /lib and /src don't reflect the changes in (the latest PR)[https://github.com/stripe/jquery.payment/commit/717257c5a75fd4d51dea110ae38d8cf35867402f]. 

[Here's](https://gist.github.com/willisplummer/7b07d7f189ee3fb232a0f2d83196e742) what I got locally — is it possible that a new version was published to bower but somehow the latest changes weren't included?
